container.te

```diff
@@ -1405,12 +1405,22 @@ fs_mounton_cgroup(container_engine_t)
 fs_unmount_cgroup(container_engine_t)
 fs_manage_cgroup_dirs(container_engine_t)
 fs_manage_cgroup_files(container_engine_t)
-fs_mount_tmpfs(container_engine_t)
 fs_write_cgroup_files(container_engine_t)
 
+allow container_engine_t cgroup_t:filesystem remount;
```

Comment:
fs_remount_cgroup(container_engine_t)

Reply:
fs_mount_all_fs(container_engine_t)
fs_remount_all_fs(container_engine_t)
fs_unmount_all_fs(container_engine_t)

Then remove all of the mount, unount and remount of file systems.

Reply:
Sadly this is not an equivalent for mounton.

Reply:
allow container_engine_t filesystem_type:{dir file} mounton;